src/Libraries/CoreNodes/List.cs

```diff
@@ -40,15 +42,267 @@ public static IList UniqueItems(IList list)
+        /// </summary>
+        /// <param name="list1">List of objects to be compared with list2.</param>
+        /// <param name="list2">List of objects to be compared with list1.</param>
+        /// <returns name="newList">The new list that contains objects that are in both List1 and List2.</returns>
+        /// <search>intersection,setintersection,set</search>
+        public static IList SetIntersection(IList<object> list1, IList<object> list2)
+        {
+            return Enumerable.Intersect(list1, list2).ToList();
+        }
+
+        /// <summary>
+        ///     Returns a new list that includes objects that are present in either List1 or List2.
+        /// </summary>
+        /// <param name="list1">List of objects to be included.</param>
+        /// <param name="list2">List of objects to be included to List1.</param>
+        /// <returns name="newList">The new list that contains objects that are either in List1 or List2.</returns>
+        /// <search>union,setunion,set</search>
+        public static IList SetUnion(IList<object> list1, IList<object> list2)
+        {
+            return Enumerable.Union(list1, list2).ToList();
+        }
+
+        /// <summary>
+        ///     Returns the index of the element in the given list.
+        /// </summary>
+        /// <param name="list">The list to find the element in.</param>
+        /// <param name="element">The element whose index is to be returned.</param>
+        /// <returns name="int">The index of the element in the list.</returns>
+        /// <search>index,indexof</search>
+        public static int IndexOf(IList list, object element)
+        {
+            return list.IndexOf(element);
+        }
+
+        /// <summary>
+        ///     Returns the number of false boolean values in the given list.
+        /// </summary>
+        /// <param name="list">The list find the false boolean values.</param>
+        /// <returns name="int">The number of false boolean values in the list.</returns>
+        /// <search>false,countfalse</search>
+        public static int CountFalse(IList list)
+        {
+            return CountBool(list, false);
+        }
+
+        /// <summary>
+        ///     Returns the number of true boolean values in the given list.
+        /// </summary>
+        /// <param name="list">The list find the true boolean values.</param>
+        /// <returns name="int">The number of true boolean values in the list.</returns>
+        /// <search>true,counttrue</search>
+        public static int CountTrue(IList list)
+        {
+            return CountBool(list, true);
+        }
+
+        /// <summary>
+        ///     Returns the flattened 1D list of the multi-dimensional input list.
+        /// </summary>
+        /// <param name="list">The list to be flattened.</param>
+        /// <returns name="list">The flattened 1D list.</returns>
+        /// <search>flatten,1D</search>
+        public static IList FlattenCompletely(IList list)
```

Comment:
Builtin `Flatten()` took no arguments.  When I made the Thunderstorm implementation of this Builtin, I mapped it to a new function `List.FlattenCompletely()` because of the existing `List.Flatten()`, rather than adding an overload.